docs/core/integration-quality-scale/iqs037.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "IQS037 - Keep the config flow understandable and validate where needed"
```

Comment:
I don't think this should be a special rule. I think this should be part of the config flow rule.